src/foraging_gui/MyFunctions.py

```diff
@@ -25,6 +25,7 @@
 class GenerateTrials():
     def __init__(self,win):
         self.win=win
+        self.B_Bias = None  # lick bias
```

Comment:
`B_Bias` should initialize as an empty list `[]`, since we append a value for each trial